api/src/test/java/com/epam/pipeline/app/AclTestConfiguration.java

```diff
@@ -214,6 +215,9 @@
     @MockBean
     protected PipelineRunDockerOperationManager pipelineRunDockerOperationManager;
 
+    @MockBean
```

Comment:
Let's move it to `AclTestBeans`.

Reply:
Class `AclTestConfiguration` configures environment for test classes which doesn't have anything in common with the pull request test classes. The change is only required because acl classes are moved between packages.